packages/sdk/src/signingcosmwasmclient.ts

```diff
@@ -150,6 +160,8 @@ export class SigningCosmWasmClient extends CosmWasmClient {
       compressedSize: compressed.length,
       compressedChecksum: Encoding.toHex(new Sha256(compressed).digest()),
       codeId: Number.parseInt(codeIdAttr.value, 10),
+      logs: result.logs,
```

Comment:
Looks like all results get normalized with logs and transactionHash. I like that